src/core/friendly_errors/fes_core.js

```diff
@@ -168,6 +176,200 @@ if (typeof IS_MINIFIED !== 'undefined') {
     console.log(translator('fes.pre', { message }));
   };
 
+  const editDistance = (w1, w2) => {
+    // An implementation of
+    // https://en.wikipedia.org/wiki/Wagner%E2%80%93Fischer_algorithm
+    // to compute the Levenshtein distance
```

Comment:
Maybe just a line to say what this is useful for -- something like 'to detect how dissimilar two strings are'.

Reply:
Yes I will add that